Patch already in upstream master:

JENT: GCD - use a sensible max
On some systems like an EPYC, the GCD is observed to be larger than 100.
As there is no reason to not allow larger GCD values, let us simply
allow it.

Signed-off-by: Stephan Mueller <smueller@chronox.de>

---

diff -aur a/src/jitterentropy-gcd.c b/src/jitterentropy-gcd.c
--- a/src/jitterentropy-gcd.c
+++ b/src/jitterentropy-gcd.c
@@ -36,6 +36,7 @@
 	/* Make a greater a than or equal b. */
 	if (a < b) {
 		uint64_t c = a;
+
 		a = b;
 		b = c;
 	}
@@ -113,12 +114,8 @@
 		goto out;
 	}
 
-	/*
-	 * Ensure that we have variations in the time stamp below 100 for at
-	 * least 10% of all checks -- on some platforms, the counter increments
-	 * in multiples of 100, but not always
-	 */
-	if (running_gcd >= 100) {
+	/* Set a sensible maximum value. */
+	if (running_gcd >= UINT32_MAX / 2) {
 		ret = ECOARSETIME;
 		goto out;
 	}
